src/toil/job.py

```diff
@@ -1722,3 +1721,185 @@ def convertPromises(kwargs):
             elif isinstance(kwargs.get(r), PromisedRequirement):
                 foundPromisedRequirement = True
         return foundPromisedRequirement
+
+
+def docker_call(job,
+                tool,
+                parameters=None,
+                work_dir='.',
+                docker_parameters=None,
+                outfile=None,
+                check_output=False,
+                defer=None):
+    """
+    Calls Docker for a particular tool with the specified parameters
+
+    Example of using docker_call in a Toil pipeline:
```

Comment:
Since this example references cgl-docker-lib indirectly via quay, might make sense (or not) to link to said repo.